target/linux/brcmiproc/bcm58800/base-files/lib/upgrade/platform.sh

```diff
@@ -0,0 +1,17 @@
+platform_check_image() {
+  # i know no way to verify the image
+  return 0;
```

Comment:
The semicolon is not needed, is it?

Reply:
Removed this file - please see comment above.